client/tailwind.config.js

```diff
@@ -112,6 +121,23 @@ module.exports = {
       });
       addComponents(scale);
     }),
+    /**
+     * CSS Custom properties adapter for font families that prepares
+     * the root custom properties.
+     * Note: intentionally not using theme as the theme will be the variable usage
+     * without the raw font family values easily accessible.
+     */
+    plugin(({ addBase }) => {
+      addBase({
+        ':root': Object.entries(fontFamily).reduce(
+          (obj, [key, value]) => ({
+            ...obj,
+            [`--w-font-${key}`]: value.join(', '),
+          }),
+          {},
+        ),
+      });
+    }),
```

Comment:
Same as above – I think it’s worth considering automating this less so the variables - values pairings are clearer, and if you search a variable’s name in the code, you can find exactly what it’s set to. This will scale much better as we’ll naturally add more CSS variables that come from different JS values, and for people who don’t know JS / Tailwind well, this will be more understandable.

What do you think?

```suggestion
    /**
     * CSS Custom properties defined from our design tokens.
     */
    plugin(({ addBase }) => {
      addBase({
        ':root': {
          '--w-font-sans': fontFamily.sans.join(', '),
          '--w-font-mono': fontFamily.mono.join(', '),
        },
      });
    }),
```